lib/eal/unix/eal_unix_timer.c

```diff
@@ -9,6 +9,10 @@
 void
 rte_delay_us_sleep(unsigned int us)
 {
```

Comment:
Do we really need this? I don't remember we support that before. I could be wrong.

Reply:
This is not in the critical path. But if we need DPDK to sleep, or some other similar thing, we might want to support this by our scheduler API.